lambda_functions/functions/monitoring/monitoring.py

```diff
@@ -0,0 +1,83 @@
+import psycopg2
+import boto3
+import os
+from botocore.exceptions import ClientError
+import json
+
+
+def lambda_handler(event, context):
+
+    checks = {
+        "queued_documents": queued_documents,
+    }
+
+    print(event)
```

Comment:
Do we want this `print`, and the `print` on line 64?

Reply:
Python can use print to log stuff regardless of log level. This first one was left in from debugging though so good spot! The other one is legit. I could use a logger set to info to do this also but this works so not sure if there's an advantage for this use case :)

Reply:
Ah okay 👍